Runtime/Model/BacktraceReport.cs

```diff
@@ -125,6 +130,24 @@ private void SetDefaultAttributes()
             }
         }
 
+        /// <summary>
+        /// Assign source code to Backtrace report - text available in the right panel in web debugger.
+        /// </summary>
+        /// <param name="text"></param>
+        internal void AssignSourceCodeToReport(string text)
+        {
+            if (DiagnosticStack == null || DiagnosticStack.Count == 0)
+            {
+                return;
+            }
+
+            SourceCode = new BacktraceSourceCode()
+            {
+                Text = text
+            };
+            // assign log information to first stack frame
+            DiagnosticStack[0].SourceCode = BacktraceSourceCode.SOURCE_CODE_PROPERTY;
```

Comment:
@konraddysput can you add the source code to EACH FRAME of the faulting thread? I know there's a ticket for this because lots of users like to have the logs for whatever frame they click on.

I can't find the ticket RN but I am sure it was customer feedback.

Reply:
It's now available for every frame.